arangod/RestHandler/RestReplicationHandler.cpp

```diff
@@ -842,6 +842,13 @@ void RestReplicationHandler::handleCommandRestoreData() {
                         res.errorMessage());
     }
   } else {
+    // if result is ok and the collection is the '_apps' service collection,
+    // we need to trigger the foxx selfHeal function in order to make them
+    // publicly available.
+    if (colName == "_apps") {
```

Comment:
See above.
```suggestion
  if (collectionName == "_appbundles" || collectionName == "_apps") {
```

Reply:
Yes, we could do it like this. But we should discuss further. If that is the right way to do it.